src/features/todos.ts

```diff
@@ -1,8 +1,27 @@
-import { createSlice } from '@reduxjs/toolkit';
 import { Todo } from '../types/Todo';
 
-export const todosSlice = createSlice({
-  name: 'todos',
-  initialState: [] as Todo[],
-  reducers: {},
+type SetTodosAction = {
+  type: 'todos/SET';
+  payload: Todo[];
+};
+
+const setTodos = (todos: Todo[]): SetTodosAction => ({
+  type: 'todos/SET',
+  payload: todos,
 });
+
+export const actions = { setTodos };
+
+type Action = SetTodosAction;
+
+const todosReducer = (action: Action, todos: Todo[] = []): Todo[] => {
```

Comment:
The `todosReducer` function has its parameters in the wrong order. According to Redux convention, the first parameter should be `state` and the second should be `action`. The correct signature should be `const todosReducer = (todos: Todo[] = [], action: Action): Todo[] => {`.